src/components/Questions.tsx

```diff
@@ -1,6 +1,6 @@
 import QuizModal from './QuizModal';
 
-const Questions = ({ currQuestion, questionNumber, totalQuestions, modalProps, chooseAnswer, points, choicesArr, checkAnswer }) => {
+const Questions: React.FC<{ currQuestion: { Question: string }, questionNumber: number, totalQuestions: number, modalProps: any, chooseAnswer: any, points: number, choicesArr: string[][], checkAnswer: Function}>= ({ currQuestion, questionNumber, totalQuestions, modalProps, chooseAnswer, points, choicesArr, checkAnswer }) => {
```

Comment:
I think this would be better for readability reasons to break up this long line into smaller lines like this:

```
const Questions: React.FC<{
  currQuestion: { Question: string };
  questionNumber: number;
  totalQuestions: number;
  modalProps: any;
  chooseAnswer: any;
  points: number;
  choicesArr: string[][];
  checkAnswer: Function;
}> = ({
  currQuestion,
  questionNumber,
  totalQuestions,
  modalProps,
  chooseAnswer,
  points,
  choicesArr,
  checkAnswer,
}) => {
```